### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by hobarrera**
Clean up unused imports.

If you're interested, I'd like to do a bit more cleanup regarding coding formating and PEP-8, would that generally be acceptable?

### CODE REVIEW

File: xandikos/access.py [hunk 1]
```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```
Comment (jelmer) [inline xandikos/access.py]: If this isn't done before importing xml.etree then we can end up with multiple different ElementTree implementations, which leads to TypeErrors. 

(we should probably have a central ElementTree import somewhere that imports from defusedxml first and is used by all other parts of the code, so these two lines aren't repeated all over the place)

Comment (hobarrera) [inline xandikos/access.py]: I don't quite understand how importing this (but not referencing it) affects the rest of the code, can you explain that a bit? (I'm not arguing, but really want to understand this).

Comment (jelmer) [inline xandikos/access.py]: defusedxml monkeypatches ElementTree, which means it's best to always import defusedxml before importing anything from xml.etree:

morgaine:~% python3  
Python 3.5.3 (default, Jan 19 2017, 14:11:04) 
[GCC 6.3.0 20170118] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from xml.etree import ElementTree as ET1
>>> from defusedxml import ElementTree
>>> from xml.etree import ElementTree as ET2
>>> 
>>> e = ET2.Element('foo')
>>> s = ET1.SubElement(e, 'bar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: SubElement() argument 1 must be xml.etree.ElementTree.Element, not Element

(this probably warrants a comment for the imports, which I omitted)

Comment (hobarrera) [inline xandikos/access.py]: Cool, I wan't aware. If it does monkey-patching, won't doing it in `xandikos.__init__.py` suffice?

Comment (hobarrera) [inline xandikos/access.py]: I added the import there with a comment. Also deleted the remaining imports and nothing crashed this time. 🌮

Comment (jelmer) [inline xandikos/access.py]: Yeah, putting it in xandikos.__init__should be sufficient.

Comment (jelmer) [inline xandikos/access.py]: Thanks!

**Comment by jelmer** [comment]
Merged, thanks!

**Pull request closed without merging**